src/components/link.rs

```diff
@@ -67,7 +68,18 @@ pub fn link<T>(props: &LinkProperties<T>) -> Html
 where
     T: Target + 'static,
 {
-    let router = use_router::<T>().expect("Need Router or Nested component");
+    info!("Render Link {:?}", props.to);
+    let router = match use_router::<T>() {
+        None => {
+            error!("No router");
```

Comment:
Please use `log::error`, like in the other cases.

Reply:
sorry, I missed that